src/Util/CliLoginHelper.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace ItkDev\OpenIdConnectBundle\Util;
+
+use ItkDev\OpenIdConnectBundle\Exception\TokenNotFoundException;
+use Symfony\Component\Uid\Uuid;
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * Helper class for CLI login.
+ */
+class CliLoginHelper
+{
+    /**
+     * @var CacheItemPoolInterface
+     */
+    private $cache;
+
+    /**
+     * @var string
+     */
+    private $itkNamespace;
+
+    public function __construct(CacheItemPoolInterface $cache)
+    {
+        $this->cache = $cache;
+        $this->itkNamespace = 'itk-dev-cli-login';
```

Comment:
Given the way this is used, a class constant would be more appropriate
```
private const ITK_NAMESPACE = 'itk-dev-cli-login';

...

$key = str_replace(self::ITK_NAMESPACE, '', $decodedKeyWithNamespace);
```

Reply:
Used `ITK_NAMESPACE` class constant.